docs/UserGuide.md

```diff
@@ -128,6 +128,22 @@ Examples:
 * `find alex david` returns `Alex Yeoh`, `David Li`<br>
   ![result for 'find alex david'](images/findAlexDavidResult.png)
 
+### Locating persons by tag : `findtag`
+
+Find persons with the given tag.
+
+Format: `findtag KEYWORD [MORE_KEYWORDS]`
+
+* The search is case-insensitive. e.g tag will match Tag
+* The order of the keywords does not matter. e.g. Tag One will match One Tag
```

Comment:
AB3's original User Guide didn't document this, but tags are one word, i.e. they cannot contain whitespaces. As a result, this comment doesn't make sense. Should be fine to just remove it. :thinking:

Reply:
`Tag` and `One` are 2 different tags in this case.

Reply:
Yep, I see. In that case, do you think it's redundant to have this point, since it seems to be covered by the point that persons matching at least one tag will be returned?

Reply:
Yup that's true. I'll resolve it. 👍